File: java/src/main/java/ModuleA.java
public class ModuleA {
    public String test() {
        return "youpii";
    }

    public String sayYes() {
        return "no";
    }

    public String openTheDoor(String key) {
        return "trash your mouse".equals(key) ? "unlock" : "locked";
    }

    public boolean theFirstVerseOfTheSongIs(String firstVerse) {
        return firstVerse.toLowerCase().contains("somewhere over the rainbow");
    }
}

File: typescript/src/ModuleA.ts
export default class ModuleA {

    test() {
        return 'youpii';
    }

    sayYes() {
        return 'no';
    }

    openTheDoor(key: string) {
        return key === 'trash your mouse' ? 'unlocked' : 'locked';
    }

    theFirstVerseOfTheSongIs(firstVerse: string): boolean {
        return firstVerse.toLowerCase().includes("somewhere over the rainbow");
    }
}
